source/stylesheets/refills/_modal.scss

```diff
@@ -38,8 +38,8 @@
 
   .modal-fade-screen { // overlay
     @include transition(opacity 0.25s ease);
-    @include position(fixed, 0px 0px 0px 0px);
-    background: rgba(0,0,0, 0.85);
+    @include position(fixed, 0);
+    background-color: rgba(#000, 0.85);
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.